### PR TITLE
lab 1: interactive field

### PR DESCRIPTION
**Pull request by YuliaDyka**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        if (index != -1) {
+          targetText += index.toString();
+        } else {
+          targetText += 'ERR';
+        }
+      }
+    });
+  }
+
+  List<String> strNumbers = [
```
Comment (DenysDoskochynskyi) [inline lib/main.dart]: Maybe better to use modifier   `static const` to optimise build

File: lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  String enteredText = '';
+  String targetText = '';
+
+  void _incrementCounter() {
+    setState(() {});
+  }
+
+  void _setText(){
+    setState(() {
+      final List<String> strList = enteredText.split(' ');
```
Comment (DenysDoskochynskyi) [inline lib/main.dart]: No need to extra duplicate type `List<String>` just remove it

File: lib/main.dart [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  TextEditingController textController = TextEditingController();
+  String enteredText = '';
+  String targetText = '';
+
+  void _incrementCounter() {
```
Comment (DenysDoskochynskyi) [inline lib/main.dart]: Why we need this method?

File: lib/main.dart [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                onPressed: _setText,
+                style: ButtonStyle(
+                    elevation: MaterialStateProperty.all(8),
+                    backgroundColor: MaterialStateProperty.all(
+                        const Color.fromARGB(255, 215, 222, 250),),),
+                child: const Text('Submit'),),
+          ],
+        ),
+      ),
+      floatingActionButton: FloatingActionButton(
```
Comment (DenysDoskochynskyi) [inline lib/main.dart]: Maybe better to remove this widget if the didn't contain any UI/UX improvements for app?

**Review by DenysDoskochynskyi — CHANGES_REQUESTED** [review]
Everything is fine, but please fix comments&use code reformatting:)

File: lib/main.dart (hunk 1 above)
Comment (YuliaDyka) [reply lib/main.dart]: Fixed, thanks

File: lib/main.dart (hunk 2 above)
Comment (YuliaDyka) [reply lib/main.dart]: Fixed, thanks

File: lib/main.dart (hunk 4 above)
Comment (YuliaDyka) [reply lib/main.dart]: Forgot to delete this widget

File: lib/main.dart (hunk 3 above)
Comment (YuliaDyka) [reply lib/main.dart]: This method was for floating action button, also forgot to delete.

**Review by DenysDoskochynskyi — APPROVED** [review]

**Pull request merged**